src/CMake/config/xrt.fp.in

```diff
@@ -0,0 +1,23 @@
+set(@PROJECT_NAME@_FOUND True)
+
+set(_export_groups @CONF_EXPORT_GROUPS@)
+set(_targets @CONF_TARGETS@)
+
+foreach(_group ${_export_groups})
+  include("${CMAKE_CURRENT_LIST_DIR}/${_group}-targets.cmake")
+endforeach()
+
+foreach(_target ${_targets})
+  get_target_property(TARGET_LOCATION @PROJECT_NAME@::${_target} LOCATION)
+  message(STATUS "Found @PROJECT_NAME@::${_target} : ${TARGET_LOCATION}")
+endforeach()
+
+get_filename_component(@PROJECT_NAME@_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+
+set(@PROJECT_NAME@_INCLUDE_DIRS "@CONF_INCLUDE_DIRS@")
+
+set(XRT_CORE_LIBRARIES @PROJECT_NAME@::xrt_core @PROJECT_NAME@::xrt_coreutil)
+set(XRT_OPENCL_LIBRARIES @PROJECT_NAME@::xilinxopencl @PROJECT_NAME@::xrt++)
+set(XRT_SWEMU_LIBRARIES @PROJECT_NAME@::xrt_swemu)
+set(XRT_HWEMU_LIBRARIES @PROJECT_NAME@::xrt_hwemu)
```

Comment:
Where are these variables used?   What purpose do they serve?

Reply:
@stsoe they are exclusively meant for the consumer of XRT. For instance:
```
find_package(XRT)
add_executable(foo foo.cpp)
target_link_libraries(foo ${XRT_CORE_LIBRARIES})
```

From what I've seen this is sort of a convention in cmake...

find_package(jsoncpp)

is expected to provide jsoncpp_INCLUDE_DIRS and jsoncpp_LIBRARIES

In our case... XRT provides many different sets of libraries... so I am doing my best to make it conform.